src/osemgrep/networking/Rule_fetching.ml

```diff
@@ -266,7 +266,7 @@ let parse_rule ~registry_caching (file : Fpath.t) :
           ~import_callback:(import_callback ~registry_caching)
           file ast
       in
-      let value_ = Eval_jsonnet.eval_program core in
+      let value_ = Eval_jsonnet.eval_program core Value_jsonnet.empty_env in
```

Comment:
Maybe we should define an Eval_jsonnet.eval_program_with_env and an Eval_jsonnet.eval_program
that calls the other one with an empty env, so we don't have to repeat ourselves everywhere by passing
this empty env. This also hide implementation details (the fact that we refactored the code to use an env instead of lazy value) and don't push this detail to the user of the API (who should not care about those details).

Reply:
Can do! We will still have to expose the one with environment in the mli though since Manifest_jsonnet_to_AST_generic has to call the one with a specified environment